core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -181,7 +189,8 @@ public RemoteLogManager(RemoteLogManagerConfig rlmConfig,
                             Time time,
                             Function<TopicPartition, Optional<UnifiedLog>> fetchLog,
                             BiConsumer<TopicPartition, Long> updateRemoteLogStartOffset,
-                            BrokerTopicStats brokerTopicStats) throws IOException {
+                            BrokerTopicStats brokerTopicStats,
+                            Metrics metrics) throws IOException {
```

Comment:
Could we add the javadoc for the new param?

Reply:
Sure, will add.